### DISCUSSION TITLE
Setting up the development environment

### DISCUSSION THREAD

**Discussion by maifeeulasad** [lanthale/PhotoSlide#18] [Q&A] [answered]
@lanthale can you share how you've set up your development. What tools are you using ?

**Comment by lanthale** [comment, marked as answer]
My development environment is:

- Netbeans 12.2
- AdoptopenJDK 11.10+9
- OSX 10.15.7 on my first laptop and Windows Surface Go with Win10 20H2 installed with the same AdoptOpenJDK release and Netbeans
- Ubuntu in a virtual machine just for checking the behavior

That is all. If you just clone the project with Netbeans everything else is setup automatically. I have not tried Eclipse or IntelliJ.

**Reply by maifeeulasad** [reply]
Can you try configuring it for IntelliJ ?

**Reply by lanthale** [reply]
I have now commited the config for intelliJ as well. Try it out if it is working correctly. At least I can now run it. I do not know if in IntelliJ it is neede to config a debug or profile session separately.

**Reply by maifeeulasad** [reply]
@lanthale tried in IntelliJ, and got this error : 
`Error: JavaFX runtime components are missing, and are required to run this application`

**Reply by lanthale** [reply]
Do you have done a clean/build before running ?

It seems that IntelliJ has not downloaded the artefacts because javafx is a dependency.
